backend/drm/drm.c

```diff
@@ -1304,8 +1304,14 @@ void scan_drm_connectors(struct wlr_drm_backend *drm) {
 				wl_list_insert(&wlr_conn->output.modes, &mode->wlr_mode.link);
 			}
 
+			bool is_mst = false;
+			drmModePropertyBlobRes *blob = drmModeGetPropertyBlob(drm->fd, wlr_conn->props.path);
```

Comment:
I had a helper function for reading blobs into a `malloc`ed buffer, and handled `prop == 0`: `get_drm_prop_blob`.

Reply:
Nice, I'll switch to that.